query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/Params.java

```diff
@@ -15,21 +16,21 @@
   private final Map<Integer, String> stringParams;
   private final Map<Integer, Float> floatParams;
   private final Map<Integer, Double> doubleParams;
-  private final Map<Integer, String> bytesStringParams;
+  private final Map<Integer, ByteString> byteStringParams;
 
   private Params(
       Map<Integer, Integer> integerParams,
       Map<Integer, Long> longParams,
       Map<Integer, String> stringParams,
       Map<Integer, Float> floatParams,
       Map<Integer, Double> doubleParams,
-      Map<Integer, String> bytesStringParams) {
+      Map<Integer, ByteString> bytesParams) {
```

Comment:
```suggestion
      Map<Integer, ByteString> byteParams) {
```

Reply:
It's indeed bytes, not a single byte. Calling it byteStringParams might be even clearer.